src/index.html

```diff
@@ -1,27 +1,485 @@
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
-  </head>
-
-  <body>
-    <h1>Catalog</h1>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
+  />
+  <title>Catalog</title>
+  <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+  />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+  />
+  <link
+    rel="stylesheet"
+    href="styles/index.scss"
+  />
+</head>
+
+<body>
+<header class="menu">
+  <div class="item logo">
+    <a href="#">
+      <img
+        src="images/logo.png"
+        alt="Logo"
+      />
+    </a>
+  </div>
+
+  <nav>
+    <ul class="menu-items">
+      <li class="item">
+        <a class="list is-active" href="#">
+          Apple
+        </a>
+      </li>
+
+      <li class="item item-3">
+        <a class="list" href="#">
+          samsung
+        </a>
+      </li>
+
+      <li class="item item-4">
+        <a class="list" href="#">
+          smartphones
+        </a>
+      </li>
+
+      <li class="item item-5">
+        <a class="list list--item-5" data-qa="nav-hover" href="#">
+          Laptops & Computers
+        </a>
+      </li>
+
+      <li class="item item-6">
+        <a class="list" href="#">
+          Gadgets
+        </a>
+      </li>
+
+      <li class="item item-7">
+        <a class="list" href="#">
+          Tablets
+        </a>
+      </li>
+
+      <li class="item item-8">
+        <a class="list" href="#">
+          Photo
+        </a>
+      </li>
+
+      <li class="item item-9">
+        <a class="list" href="#">
+          Video
+        </a>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<main class="main">
+  <section class="product-card" data-qa=«card»>
+    <div
+      class="product-card__image">
+    </div>
+
+    <div class="product-card__content">
+      <div class="content__text">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="text__code">
+          Product code: 195434
+        </p>
+      </div>
+
+      <div class="content__mark">
+        <div class="content__mark__stars">
+          <ul class="stars stars--N">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+        </div>
+
+        <div class="mark__reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="content__price">
+        <div class="price__text">
+          Price:
+        </div>
+        <div class="price__value">
+          $2,199
+        </div>
+      </div>
+
+      <div class="content__buy">
+        <a href="#" data-qa=«card-hover»>
```

Comment:
The data attribute `data-qa=«card-hover»` uses non-standard quotation marks. It should be changed to standard double quotes like `data-qa="card-hover"`.